scripts/build_wheel.py

```diff
@@ -70,13 +70,16 @@
 """
 ).lstrip()
 
-OBSOLETE_TEXT_TEMPLATE = """
+NO_LONGER_UPDATED_TEMPLATE = """
+*Note:* The `types-{distribution}` is obsolete and unmaintained.
```

Comment:
Maybe something like this? But not a blocking concern:

```suggestion
*Note:* The `types-{distribution}` is unmaintained and cannot now be updated.
```

Reply:
I tweaked your wording a bit more.

Reply:
LGTM